include/openssl/ssl.h

```diff
@@ -1416,6 +1416,8 @@ __owur const SSL_CIPHER *SSL_get_current_cipher(const SSL *s);
 __owur int SSL_CIPHER_get_bits(const SSL_CIPHER *c, int *alg_bits);
 __owur const char *SSL_CIPHER_get_version(const SSL_CIPHER *c);
 __owur const char *SSL_CIPHER_get_name(const SSL_CIPHER *c);
+__owur const char *SSL_CIPHER_standard_name(const SSL_CIPHER *c);
+__owur const char *SSL_CIPHER_ossl_name(const char *rfc_name);
```

Comment:
The SSL_CIPHER prefix usually implies that the first argument is an SSL_CIPHER object; which is not the case here. Is there some "better" name that could be used instead?